### DISCUSSION TITLE
very unstable connection while connecting an android phone to an ESP8266 running an access point

### DISCUSSION THREAD

**Discussion by Utkarsh-Deshmukh** [micropython/micropython#10910] [General]
The connection between my android phone and my esp8266 is extremely unstable when I am running it on AP mode. This doesnt happen on my PC, just on android phones. I have tried the following things and none of them have worked:
1) Powering the ESP8266 via a dedicated power source
2) updating the ESP8266 to have the latest firmware (I am using the micropython firmware `v1.19.1 (2022-06-18)`)
3) the phone and the ESP module are not very far apart.
Has anyone seen this before? (My gut feeling is that this is not an issue on the ESP side, but on the android). Any help will be much appreciated.

**Comment by Josverl** [comment]
I would check if the phone tries to 'sense' the network by checking for the ability to create dns and https requests via the AP.
I suspect that it finds it slower or less capable than it prefers, and then goes off and tries to connect to other networks or cellular. Also I have seen that some phones / people assume that common dns such as 4.4.4.4/8.8.8.8 are valid on all networks
